lib/websocket_adapter/standard.rb

```diff
@@ -0,0 +1,40 @@
+module WebsocketAdapter
+  class Standard
+    attr_reader :env, :url
+
+    def initialize(env, sock)
```

Comment:
maybe pass in `protocol` here?

Or pass in the driver?

Reply:
No please, as I described above, more protocol types will come in the future and the `new` is not a redefinition. I'm using the module in the same way as we do in `TreeView` in the UI.